app/uk/gov/hmrc/securemessage/controllers/models/generic/ConversationMetadata.scala

```diff
@@ -46,22 +48,16 @@ object ConversationMetaData {
     )
   }
 
-  private def findLatestMessage(coreConversation: Conversation): Option[Message] =
-    coreConversation.messages.sortWith(_.created.getMillis > _.created.getMillis).headOption match {
-      case Some(message) => Some(message)
-      case _             => None
-    }
+  private def findLatestMessage(coreConversation: Conversation): Message =
+    coreConversation.messages.sortBy(_.created.getMillis).reverse.head
```

Comment:
Even if its nonEmptyList is it still safe to use head ?

Reply:
That is the whole point of using NEL @muralishub - the head is no longer an `Option`

Reply:
Thanks